Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -601,6 +601,8 @@ typedef NS_OPTIONS(NSUInteger, WKDisplayCaptureSurfaces) {
 // This property is KVO compliant.
 @property (nonatomic, readonly) _WKWebProcessState _webProcessState WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
 
+- (void)_fetchSessionStorageData:(WK_SWIFT_UI_ACTOR void (^)(NSData *))completionHandler WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionOS(WK_XROS_TBA));
```

Comment:
What happened to the generic API that takes in a NSSet of types, like we did for the WKWebsiteDataStore?

This doesn't look like this can become API?

Reply:
Just brought this back as discussed.